### PR TITLE
Rewrite to python module

### PR DESCRIPTION
**Pull request by smeinecke**
- use logging module instead of print
- return results of api function calls
- only do sample code if run as program instead of loaded as module
- removed unused variables + urllib.parse

### CODE REVIEW

File: oprah-proxy.py [hunk 1]
```diff
@@ -2,19 +2,18 @@
...
-    print('DEBUG: Please install Requests for Python 3, see '
-          'http://docs.python-requests.org/en/master/user/install/ or use your '
-          'favorite package manager (e.g. apt-get install python3-requests)')
+    logging.error('Cannot import requests')
+    logging.debug('Please install Requests for Python 3, see '
+                  'http://docs.python-requests.org/en/master/user/install/ or use your '
+                  'favorite package manager (e.g. apt-get install python3-requests)')
     exit(2)
 
-
```
Comment (spaze) [inline oprah-proxy.py]: Looks like [PEP 8 requires 2 blank lines](https://www.python.org/dev/peps/pep-0008/#blank-lines) here, can you put it back?

File: oprah-proxy.py [hunk 2]
```diff
@@ -40,127 +38,129 @@ def post(self, url, data):
...
-
-you_get_a_proxy = OprahProxy('se0310',
-                             'AE4CA57D1E3C0E6711C53416BFA0988F08D41B428D26D053A4C46EC72A79B9E7')
-you_get_a_proxy.you_get_a_proxy()
-you_get_a_proxy.everybody_gets_a_proxy()
+                logging.info('Proxy in %s/%s %s:%s' %
+                             (ip['geo']['country_code'], ip['geo']['state_code'],
+                              ip['ip'], port))
+                proxies.append((ip['ip'], port, ip['geo']['country_code'], ip['geo']['state_code']))
```
Comment (spaze) [inline oprah-proxy.py]: Just an idea, what about a dictionary instead of a tuple?

**Review by spaze — COMMENTED** [review]
I like it, thanks! I have just two minor questions before merging it, could you please check them? Thanks!

**Comment by smeinecke** [comment]
Yes, sure 👍

**Comment by spaze** [comment]
Thank you, good job. I've squashed the two commits and it's now merged in a42dba1364fe3b7b5b261f52371837d91b32b91e

**Pull request closed without merging**